bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java

```diff
@@ -1593,6 +1593,11 @@ void pollCameraRunnable() {
                     sendHttpGET("/api.cgi?cmd=GetMdState&channel=" + cameraConfig.getNvrChannel() + reolinkAuth);
                 } else {
                     onvifCamera.sendOnvifRequest(RequestType.Renew, onvifCamera.subscriptionXAddr);
+                    if (onvifCamera.pullMessageRequests.intValue() == 0) {
+                        logger.info("The alarm stream was not running for Reolink camera {}, re-starting it now",
```

Comment:
```suggestion
                        logger.debug("The alarm stream was not running for Reolink camera {}, re-starting it now",
```

I suppose this should be debug (as this message is hardly of any use to the user) or warn (if the users needs to handle it, but that does not seem to be the case)